.gitignore

```diff
@@ -22,6 +22,8 @@ charts/redpanda/ci/21-eks-tiered-storage-with-creds-values.yaml
 # Dep Chart Lock file
 Chart.lock
 
+charts/redpanda/local.yaml
```

Comment:
I just added a `local.yaml` so that I could test out different configurations of the chart easily manually. I can remove this from the `.gitignore`, but it might be nice to have a file ignored so that we have a pattern around "just add this file name and you can do some manual installs with a custom values file" (so you don't have to pass a bunch of --set flags if you're trying a fairly non-default configuration).

Reply:
I understand now the purpose of this file. If possible your comment could go into .gitignore as a comment for future me :)